R/plot_taxa.R

```diff
@@ -1236,51 +1236,72 @@ plot_taxa_abund <- function(data,
 
 
 
-#' Plot sites on US map
+#' Work-in-progress upgraded plot_sites
 #'
 #' @param data (list or tbl_df, tbl, data.frame) The dataset object returned by \code{read_data()}, a named list of tables containing the observation and taxon tables, or a flat table containing columns of the observation and location tables.
 #' @param id (character) Identifier of dataset to be used in plot subtitles. Is automatically assigned when \code{data} is a dataset object containing the \code{id} field, or is a table containing the package_id column.
 #' @param alpha (numeric) Alpha-transparency scale of data points. Useful when many data points overlap. Allowed values are between 0 and 1, where 1 is 100\% opaque. Default is 1.
-#' @param labels (logical) Argument to show labels of each US state. Default is TRUE.
```

Comment:
@sokole @RafaelRangel0 removing this parameter creates a backwards compatibility breaking change. See below.

Reply:
We can add it back in if desired.

Reply:
added back, but `labels` doesn't appear to have actually done anything in the old code.